src/main/resources/zeebe_release.bpmn

```diff
@@ -281,97 +283,58 @@
       </bpmn:sequenceFlow>
       <bpmn:sequenceFlow id="Flow_06yz81p" name="No" sourceRef="Gateway_0pprots" targetRef="Event_09hnipm" />
     </bpmn:subProcess>
+    <bpmn:subProcess id="Activity_0caepsu" name="Release Candidate" triggeredByEvent="true">
+      <bpmn:endEvent id="Event_0hfv9hy">
+        <bpmn:incoming>Flow_1a33fza</bpmn:incoming>
+      </bpmn:endEvent>
+      <bpmn:startEvent id="Event_072g6mg" name="Build release candidate" isInterrupting="false">
+        <bpmn:outgoing>Flow_0laqkqg</bpmn:outgoing>
+        <bpmn:messageEventDefinition id="MessageEventDefinition_0zmenc7" />
+      </bpmn:startEvent>
+      <bpmn:sequenceFlow id="Flow_0laqkqg" sourceRef="Event_072g6mg" targetRef="Activity_1wtaxv3" />
+      <bpmn:callActivity id="Activity_1wtaxv3" name="Build release candidate">
+        <bpmn:extensionElements>
+          <zeebe:calledElement processId="build-release-candidate" propagateAllChildVariables="false" />
+        </bpmn:extensionElements>
+        <bpmn:incoming>Flow_0laqkqg</bpmn:incoming>
+        <bpmn:outgoing>Flow_1a33fza</bpmn:outgoing>
+      </bpmn:callActivity>
+      <bpmn:sequenceFlow id="Flow_1a33fza" sourceRef="Activity_1wtaxv3" targetRef="Event_0hfv9hy" />
+      <bpmn:textAnnotation id="TextAnnotation_1oejvef">
+        <bpmn:text>correlationKey: release-branch-name
+messageName: rc</bpmn:text>
+      </bpmn:textAnnotation>
+      <bpmn:association id="Association_1b25zkd" sourceRef="Event_072g6mg" targetRef="TextAnnotation_1oejvef" />
+    </bpmn:subProcess>
+    <bpmn:sequenceFlow id="Flow_0c32sfk" sourceRef="Gateway_113spke" targetRef="Activity_175is1m" />
+    <bpmn:callActivity id="Activity_175is1m" name="Build release candidate">
+      <bpmn:extensionElements>
+        <zeebe:calledElement processId="build-release-candidate" propagateAllChildVariables="false" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0c32sfk</bpmn:incoming>
+      <bpmn:outgoing>Flow_1t4de6j</bpmn:outgoing>
+    </bpmn:callActivity>
+    <bpmn:sequenceFlow id="Flow_1t4de6j" sourceRef="Activity_175is1m" targetRef="Gateway_038v8ud" />
     <bpmn:textAnnotation id="TextAnnotation_15g9k3f">
       <bpmn:text>req: release_version: string</bpmn:text>
     </bpmn:textAnnotation>
     <bpmn:association id="Association_0uafnuq" sourceRef="StartEvent_1" targetRef="TextAnnotation_15g9k3f" />
+    <bpmn:textAnnotation id="TextAnnotation_149oh6t">
+      <bpmn:text>Used by other teams to test against and by the e2e tests.</bpmn:text>
+    </bpmn:textAnnotation>
+    <bpmn:association id="Association_10w5l9m" sourceRef="Activity_175is1m" targetRef="TextAnnotation_149oh6t" />
   </bpmn:process>
+  <bpmn:message id="Message_36527rk" name="rc">
+    <bpmn:extensionElements>
+      <zeebe:subscription correlationKey="=release_branch_name" />
```

Comment:
:x: The variable `release_branch_name` is not defined when the process instance is created. As a result, an incident is created. 

The variable is created when completing the form of the task `Create release branch`.